test/e2e/utils/api/index.js

```diff
@@ -1,7 +1,7 @@
 export { closeBrowser, openBrowser } from './browser'
 export { button } from './button'
 export { click, clickParent } from './click'
-export { expectExists } from './expect'
+export { expectExists, getElement } from './expect'
```

Comment:
why is getElement exported from expect? 
add a new file element ?

Reply:
agree 👍